config/vufind/FeedbackForms.yaml

```diff
@@ -150,7 +150,7 @@ forms:
   # Main Feedback form
 
   FeedbackSite:
-    title: Send us your feedback!
+    title: Send us your feedback
```

Comment:
Should this be revised to something like feedback_title to match?

Reply:
Same here, this should become `feedback_forms_title`...

Reply:
I proposed `feedback_title` because we already have a set of strings starting with just `feedback_` and I think this fits into the same family. You do make a good point that this might make sense as a text domain, but given that the precedent is already set, I don't think we should worry about refactoring it at this time.